spec/features/admins/merchants/index_spec.rb

```diff
@@ -20,10 +24,64 @@
       expect(page).to have_link(merchant_1.name)
       expect(page).to have_link(merchant_2.name)
       expect(page).to have_link(merchant_3.name)
+      expect(page).to have_link(merchant_4.name)
+      expect(page).to have_link(merchant_5.name)
 
       click_link(merchant_1.name)
 
       expect(current_path).to eq(admin_merchant_path(merchant_1))
+
+      visit admin_merchants_path
+
+      click_link(merchant_2.name)
+
+      expect(current_path).to eq(admin_merchant_path(merchant_2))
     end
+    
+    describe 'enable and disable' do
+      it 'has an enable or disable button next to each merchant' do
+        visit admin_merchants_path
+
+        within "#merchant_#{merchant_1.id}" do 
+          expect(page).to have_button "Enable Merchant"
+          expect(page).to_not have_button "Disable Merchant"
+        end 
+
+        within "#merchant_#{merchant_2.id}" do 
+          expect(page).to have_button "Enable Merchant"
+        end 
+
+        within "#merchant_#{merchant_3.id}" do 
+          expect(page).to have_button "Enable Merchant"
+        end 
+       
+        within "#merchant_#{merchant_4.id}" do 
+          expect(page).to have_button "Disable Merchant"
+          expect(page).to_not have_button "Enable Merchant"
+        end 
+
+        within "#merchant_#{merchant_5.id}" do 
+          expect(page).to have_button "Disable Merchant"
+        end 
+      end
+
+      it "will update a merchant's status" do
+        visit admin_merchants_path
+
+        expect(merchant_1.status).to eq('disabled')
+        expect(merchant_5.status).to eq('enabled')
+
+        click_button "Disable #{merchant_5.name}"
+
+        expect(current_path).to eq admin_merchants_path
+        expect(Merchant.find(merchant_5.id).status).to eq('disabled')
+
```

Comment:
Thomas and I talked about this weird expect statement on line 77 and will bring it to our check in tomorrow with Chris